lightwood/encoder/audio/mfcc.py

```diff
@@ -9,20 +9,40 @@
 
 class MFCCEncoder(BaseEncoder):
     """
-    Audio encoder. Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing. The features are a 2D array, flattened into a 1D one.
+    Audio encoder.
+
+    Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing.
+
+    The output feature for any given audio file is a 2D array, flattened into a 1D one to comply with the expected format in lightwood mixers.
+
+    Note that this encoder does not have a .decode() method. As such, models that predict audio as output are not supported at this time.
     """  # noqa
     is_trainable_encoder: bool = False
 
     def __init__(self, is_target: bool = False):
+        """
+        :param is_target: whether this encoder's column is the target. Should be false as encoder is not bi-directional. 
+        """  # noqa
+        assert not is_target
         super().__init__(is_target)
 
     def prepare(self, priming_data: pd.Series):
+        """
+        Retrieves and stores the expected output_size.
+
+        :param priming_data: training data
+        """
         self.is_prepared = True
         priming_data = list(priming_data)
         ele = self.encode([str(priming_data[0])])[0]
```

Comment:
do we have a guarantee that the first example will always yield the same size? I don't know enough about this encoder to say either way.

Reply:
I am fairly confident this is the case, as the method in `encode()` is essentially a call to `librosa.feature.mfcc()` with the same arguments every time.